changelog/638.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:feature
```

Comment:
Features are generally large changes... Something like building a role system for intermediate CAs or adding ACME support or something similar. I think this is better suited as an `improvement`. :-)